packages/components/src/composite/v2.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Composite is a component that may contain navigable items represented by
+ * CompositeItem. It's inspired by the WAI-ARIA Composite Role and implements
+ * all the keyboard navigation mechanisms to ensure that there's only one
+ * tab stop for the whole Composite element. This means that it can behave as
+ * a roving tabindex or aria-activedescendant container.
+ *
+ * @see https://ariakit.org/components/composite
+ */
+export {
+	Composite,
+	CompositeGroup,
```

Comment:
Yes, I suppose we should. I was erring on not exporting things not in use, but that's a bit harder to argue when there's been no option to do so in the past!

Reply:
apart from `CompositeGroupLabel`, I wonder if we should export any other composite -related components ?

Reply:
The other missing components are [`CompositeProvider`](https://ariakit.org/reference/composite-provider)[^1], [`CompositeHover`](https://ariakit.org/reference/composite-hover)[^2], and [`CompositeSeparator`](https://ariakit.org/reference/composite-separator)[^3].

I'm not sure if there's much use for them (yet), but there's also not so many that it would be totally ridiculous to just include them. Open to suggestions, but I'm inclined to leave them until they're needed. Agreed that we should include [`CompositeGroupLabel`](https://ariakit.org/reference/composite-group-label) though, as we're including [`CompositeGroup`](https://ariakit.org/reference/composite-group).

[^1]: "Provides a composite store to `CompositeItem` components."
[^2]: "Renders an element in a composite widget that receives focus on mouse move and loses focus to the composite base element on mouse leave..."
[^3]: "Renders a separator for composite items."

Reply:
Added `CompositeGroupLabel`. If you think the others should be added, I will do so, but otherwise I'm leaving them for now.

Reply:
Sounds good, let's add them if/when needed